customdomain-terraform/modules/customdomain/main.tf

```diff
@@ -0,0 +1,51 @@
+variable "zone_id" {
+  type = "string"
+}
+
+variable "govuk_domain" {
+  type = "string"
+}
+
+variable "govsvcuk_domain" {
+  type = "string"
+}
+
+resource "aws_acm_certificate" "cert" {
+  domain_name       = var.govuk_domain
+  validation_method = "DNS"
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+
+resource "aws_cloudfront_distribution" "cloudfront_dist" {
+  enabled             = true
+  is_ipv6_enabled     = true
+  price_class         = "PriceClass_100" # EU and NA optimised, lower cost
+
```

Comment:
TODO: should we have `aliases = [var.govuk_domain]` here?

Reply:
If I'm interpreting [this](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/CNAMEs.html) correctly, I think the answer is "yes".